main.js

```diff
@@ -18,8 +18,11 @@
  *        MIT License. All code unless otherwise specified is
  *        Copyright (c) Matthew Knox and Contributors 2015.
  */
-
+'use strict';
 // Load NodeJS Modifications/Variables
+require('babel-core');
+require('babel-register');
+require('babel-polyfill');
```

Comment:
the only things that should ever be require()'ed here are things that will never be changed by /update (note prototypes.js is reloaded elsewhere)